docs/excel rows.md

```diff
@@ -23,13 +23,64 @@ sheet are treated differently than Rows 4 - ∞.
   - Column with key, value, and unit combinations for
   - The condition under which the property was discerned
 - <span class="row-1">identifier</span>
-  - Column with key-value pairs for material identifiers
+  - Column that can identify a row
+    - Example in everyday life can be `social security` that can identify a single citizen
```

Comment:
I don't think this sentence is necessary.

Reply:
I will remove lines 26 and 27 as chemists already know what material identifiers are